### PR TITLE
feat: Retrieve users of projects

### LINKED ISSUE #81: Cannot retrieve users of projects
`armadillo.list_projects` lists all project names, but it would be really useful to be able to list the users of these projects as well, or even get the users of a specific project:
``` R
get_projects_info <- function() {
  response <- httr::GET(
    url = armadillo_url,
    path = "/access/projects",
    config = httr::add_headers(auth_header)
  )
  .handle_request_error(response)
  return(httr::content(response, as = "parsed"))
}
```

``` R
library(rlist)
get_project_users <- function(projects_info, project_name) {
  return(list.filter(projects_info, name == project_name)[[1]]$users)
}
```

### PR DESCRIPTION
**Pull request by SietsmaRJ**
- Added getter function "get_projects_info()" to utilize endpoint "/access/projects" in a better way.
- Added getter function "get_project_users(project_name)" to get all users from a given project name.
- Added testing
- Added documentation
- Updated CI

Closes #81

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
     - run:
         name: Install R deps takes 15min, but no worries we skipinstalled from cache when this PR has installed them before
         command: |
-          install2.r --skipinstalled --repo https://cloud.r-project.org devtools pkgdown markdown rmarkdown mockery webmockr httr urltools xml2 arrow MolgenisAuth DSMolgenisArmadillo git2r ellipsis vctrs covr desc
+          install2.r --skipinstalled --repo https://cloud.r-project.org devtools pkgdown markdown rmarkdown mockery webmockr httr urltools xml2 arrow MolgenisAuth DSMolgenisArmadillo git2r ellipsis vctrs covr descb rlist
```
Comment (clemens-tolboom) [inline .circleci/config.yml:49]: Is `desc` dependency `descb` meant to change?

File: tests/testthat/test-project.R [hunk 2]
```diff
@@ -81,3 +81,97 @@ test_that("armadillo.delete_project deletes project", {
...
+        {
+          \"name\": \"lifecycle\",
+          \"users\": []
+        },
+        {
+          \"name\": \"other-project\",
+          \"users\": [\"john\", \"tommy\"]
+        }
+      ]",
+      headers = list("Content-Type" = "application/json")
```
Comment (clemens-tolboom) [inline tests/testthat/test-project.R:103]: Can we use single quotes for readability?
```
      body = '[
        {
          "name": "lifecycle",
          "users": []
        },
        {
          "name": "other-project",
          "users": ["john", "tommy"]
        }
      ]',
```

File: tests/testthat/test-project.R [hunk 3]
```diff
@@ -81,3 +81,97 @@ test_that("armadillo.delete_project deletes project", {
...
+      body = "[
+        {
+          \"name\": \"lifecycle\",
+          \"users\": []
+        },
+        {
+          \"name\": \"other-project\",
+          \"users\": [\"john\", \"tommy\"]
+        }
+      ]",
```
Comment (clemens-tolboom) [inline tests/testthat/test-project.R:124]: Is this the same data ... DRY ? Can we make it a common variable?

Else : please remove the escaping.

**Comment by clemens-tolboom** [comment]
Building gives 1 note

> N  checking R code for possible problems (2.7s)
   armadillo.get_project_users: no visible binding for global variable
     ‘name’
   Undefined global functions or variables:
     name

**Review by marikaris — APPROVED** [review]

**Pull request merged**